src/learner/tests/test_learner.py

```diff
@@ -6,6 +6,8 @@
 from learner.learner import UserDocumentsAccumulator, UserData, UserDoc, _similarity_by_row, _FixedSizeHeap
 
 # test implementation of L2 norm without re-using numpy code
+
```

Comment:
Why adding two blank lines ? am I missing something ? The above comment is not related to norm function ?